account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -45,9 +46,33 @@ def default_get(self, field_list):
     def extend_payment_order_domain(self, payment_order, domain):
         if payment_order.payment_order_type == 'payment':
             domain += [('account_id.type', 'in', ('payable', 'receivable')),
-                       ('amount_to_pay', '>', 0)]
+                       ('credit', '>', 0)]
         return True
 
+    @api.model
+    def filter_lines(self, lines):
+        """ Filter move lines before proposing them for inclusion
+            in the payment order.
+
+        This implementation filters out move lines that are already
+        included in draft or open payment orders. This prevents the
+        user to include the same line in two different open payment
+        orders. When the payment order is sent, it is assumed that
+        the move will be reconciled soon (or immediately with
+        account_banking_payment_transfer), so it will not be
+        proposed anymore for payment.
+
+        See also https://github.com/OCA/bank-payment/issues/93.
+
+        :param lines: recordset of move lines
+        :returns: list of move line ids
+        """
+        payment_lines = self.env['payment.line'].\
+            search([('order_id.state', 'in', ('draft', 'open')),
+                    ('move_line_id', 'in', lines.ids)])
+        to_exclude = set([l.move_line_id.id for l in payment_lines])
+        return [l.id for l in lines if l.id not in to_exclude]
```

Comment:
@pedrobaeza I considered set difference but I wanted to preserve the order of the lines, and since volumes will never be huge this should be ok?

Two questions:
* Do you know if recordset difference preserves order? In which case I will adapt the code.
* What do you mean by "I think you can also assign directly the recordset to the field"?